src/cares_wrap.cc

```diff
@@ -1869,21 +1871,20 @@ void AfterGetAddrInfo(uv_getaddrinfo_t* req, int status, struct addrinfo* res) {
   uv_freeaddrinfo(res);
```

Comment:
Fwiw, this might also be something we’d want to use RAII for in some way (smart pointer or `OnScopeLeave`)?

Reply:
I'll take a closer look at this in a follow up PR. Thanks